cmd/bridge/main.go

```diff
@@ -204,3 +223,50 @@ func attachFileLogger(log logger.Logger, flagsConfig *relay.ContextFlagsConfig)
 
 	return fileLogging, nil
 }
+
+func buildNetMessenger(cfg relay.Config, marshalizer marshal.Marshalizer) (relayp2p.NetMessenger, error) {
```

Comment:
could have been a relayer factory :smile:

Reply:
soon :TM:
We will end-up creating factories that can be easily tested, of course.